test-app/src/main/kotlin/uk/gov/justice/digital/hmpps/testapp/TestApp.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.testapp
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class TestApp
```

Comment:
Added a module with a copy of the latest hmpps-template-kotlin so we can write integration where we consume the library.